libcovebods/lib/common_checks.py

```diff
@@ -40,15 +86,18 @@ def __init__(self, json_data):
         self.person_statements_seen_in_ownership_or_control_statement = []
         self.entity_statements_seen = []
         self.entity_statements_seen_in_ownership_or_control_statement = []
+        self.ownership_or_control_statements_seen = []
```

Comment:
`self.ownership_or_control_statements_seen` and `self.statement_ids_counted` are declared in more than one place, is this deliberate?

Reply:
Yeah, Pep complained if I didn't do it in init, and I wanted it in Run incase someone called run twice on the same object. Maybe my use of the "run" function is a bit odd here, happy to discuss.